aten/src/ATen/native/Resize.cpp

```diff
@@ -10,4 +10,8 @@ Tensor& resize_cpu_(Tensor& self, IntArrayRef size) {
   return self;
 }
 
+Tensor& resize_as_cpu_(Tensor& self, const Tensor& the_template) {
```

Comment:
I know that its not your change but im wondering why this file isn't located in native/cpu to follow the cuda example.

Reply:
native/cpu compiles 3 times for old CPU, old with AVX and AVX2. This code cannot be optimized with AVX